crates/bevy_input/src/lib.rs

```diff
@@ -74,6 +74,7 @@ impl Plugin for InputPlugin {
             // gamepad
             .add_event::<GamepadConnectionEvent>()
             .add_event::<GamepadButtonChangedEvent>()
+            .add_event::<GamepadButtonInput>()
```

Comment:
I don't like it, but almost all other events have their name ending with `Event`.

Could you either rename that new event to `GamepadButtonInputEvent`, or renamed all the others?

Reply:
`GamepadButtonInput` was chosen to match the naming of `KeyboardInput`; I agree the inconsistent naming isn't great, so it might be good to rename both of them to end with `Event`.